legend-depot-server/src/main/java/org/finos/legend/depot/server/resources/dependencies/DependenciesResource.java

```diff
@@ -93,14 +96,14 @@ public ProjectDependencyReport analyzeDependencyTree(@ApiParam("projectDependenc
     @Path("/projects/{groupId}/{artifactId}/versions/{versionId}/dependantProjects")
     @ApiOperation(GET_DEPENDANT_PROJECTS)
     @Produces(MediaType.APPLICATION_JSON)
-    public List<ProjectVersionPlatformDependency> getDependantProjects(@PathParam("groupId") String groupId,
-                                                                       @PathParam("artifactId") String artifactId,
-                                                                       @PathParam("versionId") @ApiParam(value = VersionValidator.VALID_VERSION_ID_TXT) String versionId,
-                                                                       @QueryParam("latestOnly") @DefaultValue("false")
-                                                                       @ApiParam("Whether to only return the latest version of dependant projects") boolean latestOnly
+    public Response getDependantProjects(@PathParam("groupId") String groupId,
+                                                                         @PathParam("artifactId") String artifactId,
+                                                                         @PathParam("versionId") @ApiParam(value = VersionValidator.VALID_VERSION_ID_TXT) String versionId,
+                                                                         @QueryParam("latestOnly") @DefaultValue("false") @ApiParam("Whether to only return the latest version of dependant projects") boolean latestOnly,
+                                                                         @Context Request request
     )
     {
-        return handle(GET_DEPENDANT_PROJECTS, GET_DEPENDANT_PROJECTS + groupId + artifactId, () -> transform(this.projectApi.getDependantProjects(groupId, artifactId, versionId, latestOnly)));
+        return handle(GET_DEPENDANT_PROJECTS, GET_DEPENDANT_PROJECTS + groupId + artifactId, () -> transform(this.projectApi.getDependantProjects(groupId, artifactId, versionId, latestOnly)), request, groupId, artifactId, versionId);
```

Comment:
wjat ab out latestOnly...again refer to my comment about params

Reply:
This API cannot have etag on it (in this manner) as the API response is not static for any given argument set. Only APIs where the response can never change for the given GAV should be in scope here (with a constant to allow cache busting as mentioned previously)